src/Psy/ExecutionLoop/ForkingLoop.php

```diff
@@ -166,6 +166,8 @@ private function serializeReturn(array $return)
                 $serializable[$key] = $value;
             } catch (\Exception $e) {
                 // we'll just ignore this one...
+            } catch (\Error $e) {
```

Comment:
Yeah, plus in the regular `Loop` class it catches `Error` too. @bobthecow Do you want me to change it to throwable?